packages/webui/src/client/ui/Settings/Studio/PackageManager/AccessorTableRow.tsx

```diff
@@ -411,14 +412,21 @@ export function AccessorTableRow({
 										itemKey={`container.accessors.${accessorId}.ISAUrls`}
 										overrideHelper={overrideHelper}
 									>
-										{(value, handleUpdate) => (
-											<TextInputControl
-												modifiedClassName="bghl"
-												classNames="input text-input input-l"
-												value={value}
-												handleUpdate={handleUpdate}
-											/>
-										)}
+										{(value, handleUpdate) => {
+											// Convert array of strings into comma-separated string for the input:
+											const strValue = Array.isArray(value) ? value.join(', ') : value
```

Comment:
Is there a specific reason for using a comma separated string here instead of the `MultiLineTextInputControl` component?

Reply:
Not really. It was rendered as a comma-separated string [before in R51](https://github.com/Sofie-Automation/sofie-core/blob/release51/meteor/client/ui/Settings/Studio/PackageManager.tsx#L644), so I just preserved the existing functionality.